keanu-project/src/main/java/io/improbable/keanu/distributions/discrete/Binomial.java

```diff
@@ -8,24 +8,34 @@
 import org.apache.commons.math3.util.CombinatoricsUtils;
 import org.nd4j.linalg.util.ArrayUtil;
 
+/**
+ * @see "Computer Generation of Statistical Distributions
+ * by Richard Saucier
+ * ARL-TR-2168 March 2000
+ * 5.2.2 page 43"
+ */
 public class Binomial implements DiscreteDistribution {
 
-    private final DoubleTensor p;
-    private final IntegerTensor n;
+    private final DoubleTensor successProbability;
+    private final IntegerTensor numberOfTrials;
```

Comment:
sgtm

I'd also expect to see them in the other order (it's always `B(n,p)` as far as I know)